boa/src/syntax/lexer/lexer_old.rs

```diff
@@ -477,7 +477,7 @@ impl<'a> Lexer<'a> {
     }
 
     /// Runs the lexer until completion, returning a [LexerError] if there's a syntax issue, or an empty unit result
-    ///
+    //
```

Comment:
This change should be reverted.

Reply:
Yep, will revert.

Reply:
Have deleted the file as it is no longer needed (it is in the github history).